optuna/visualization/_param_importances.py

```diff
@@ -24,6 +25,64 @@
 logger = get_logger(__name__)
 
 
+class _ImportancesInfo(NamedTuple):
+    title: str
+    x_axis_name: str
+    y_axis_name: str
+    importance_values: List[float]
+    param_names: List[str]
+    importance_labels: List[str]
+
+
+def _get_importances_info(
+    study: Study,
+    evaluator: Optional[BaseImportanceEvaluator],
+    params: Optional[List[str]],
+    target: Optional[Callable[[FrozenTrial], float]],
+    target_name: str,
+) -> _ImportancesInfo:
+    _check_plot_args(study, target, target_name)
+
+    title = "Hyperparameter Importances"
+    x_axis_name = f"Importance for {target_name}"
+    y_axis_name = "Hyperparameter"
+
+    # Importances cannot be evaluated without completed trials.
+    # Return an empty figure for consistency with other visualization functions.
+    trials = _filter_nonfinite(
+        study.get_trials(deepcopy=False, states=(TrialState.COMPLETE,)), target=target
+    )
+
+    if len(trials) == 0:
+        logger.warning("Study instance does not contain completed trials.")
+        return _ImportancesInfo(
+            title=title,
+            x_axis_name=x_axis_name,
+            y_axis_name=y_axis_name,
+            importance_values=[],
+            param_names=[],
+            importance_labels=[],
```

Comment:
Similarly to EDF plot - maybe consider those attributes as `Optional`?

Reply:
I suppose so.